### PR TITLE
changing php vagrant version to 5.6

### PR DESCRIPTION
**Pull request by teiling88**

### CODE REVIEW

File: provision.sh [hunk 1]
```diff
@@ -1,7 +1,7 @@
 #!/usr/bin/env bash
 
 # enter url for SQL dump. e.g. http://opencaching.de/dump.sql
-DUMP_URL=""
+DUMP_URL="http://cdn.opencaching.de/opencaching_dump.sql.gz"
```
Comment (mirsch) [inline provision.sh]: is this public now?

Comment (teiling88) [inline provision.sh]: No sorry i removed it already

**Comment by following5** [comment]
@mirsch @teiling88 
I have removed your comments so that the non-public URL is no longer visible here (copyright issue).

**Comment by mirsch** [comment]
👍

**Pull request merged**